modules/gltf/editor/editor_scene_importer_blend.cpp

```diff
@@ -152,9 +222,17 @@ Node *EditorSceneFormatImporterBlend::import_scene(const String &p_path, uint32_
 		parameters_map["export_tangents"] = false;
 	}
 	if (p_options.has(SNAME("blender/animation/group_tracks")) && p_options[SNAME("blender/animation/group_tracks")]) {
-		parameters_map["export_nla_strips"] = true;
+		if (major_version >= 3 and minor_version >= 6) {
```

Comment:
```suggestion
		if (major_version >= 3 && minor_version >= 6) {
```

Reply:
Also this will require that the version is either 3.6 or greater, or 4.6 or greater, etc.